src/style.css

```diff
@@ -1 +1,78 @@
-/* Styles go here */
+html {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  margin: 0;
+  padding: 0 50px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  padding: 0;
+  margin: 0;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  display: flex;
+  text-transform: uppercase;
+  width: 100%;
+}
+
+.nav__item {
+  padding: 0;
+  margin: 0;
+  margin-right: 20px;
+}
+
+.nav__item:last-child {
+  margin: 0;
+}
+
+.nav__link {
+  display: flex;
+  text-decoration: none;
+  color: #000;
+  align-items: center;
+  height: 60px;
```

Comment:
don't hardcode height, use paddings and display: inline-block;

Reply:
Hardcoded height due to my using of display: flex. Using flex prevented from correctly centering items with padding. When paddings where removed and hard height was added (according to figma design) the items correctly centered themselves.
This change was suggested by Illia on my QnA session yesterday. 
Is it better to use display: inline-block with paddings instead of display: flex with hard height in future situations with navigation (similar to this task)?